static/app/views/dashboards/widgets/bigNumberWidget/bigNumberWidgetVisualization.tsx

```diff
@@ -79,15 +79,22 @@ export function BigNumberWidgetVisualization(props: BigNumberWidgetVisualization
   return (
     <Wrapper>
       <NumberAndDifferenceContainer>
-        {props.thresholds && (
-          <ThresholdsIndicator
-            preferredPolarity={props.preferredPolarity}
-            thresholds={props.thresholds}
-            unit={unit ?? ''}
-            value={clampedValue}
-            type={type ?? 'integer'}
-          />
-        )}
+        {defined(props.thresholds?.max_values.max1) &&
```

Comment:
```suggestion
        {defined(props.thresholds?.max_values[ThresholdMaxKeys.MAX_1]) &&
```
nit: this maybe feels a little cleaner to me, although more verbose. not sure, what do you think?